app/static/lib/speed.js

```diff
@@ -273,23 +279,23 @@ function readSection(pageNo, spdScore, breaks, countingMode) {
         // console.info('clefList: ', clefList);
         for (let clef of clefList) { // check clefs of measure, ignore @sameas
           if (clef.getAttribute('sameas')) continue;
-          if (countingMode === 'encodedBreaks' && breaks.includes(clef.nodeName))
-            break;
-          let staff = clef.closest('staff, staffDef');
-          let staffNo = -1;
-          if (staff) staffNo = staff.getAttribute('n');
-          else continue;
+          const staffNo = getStaffNumber(clef);
+          if (!staffNo) continue;
           // console.info('clefList stffNo: ' + stffNo);
-          let staffDef = findByAttributeValue(spdScore, 'n', staffNo, 'staffDef');
+          let staffDef = spdScore.querySelector(`staffDef[n=${staffNo}]`);
```

Comment:
Here it crashes, double quotes missing for `@n`: should be ``spdScore.querySelector(`staffDef[n="${staffNo}"]`);``